examples/data_apis/tensorflow_data_api_tiledb_sparse.ipynb

```diff
@@ -279,7 +287,7 @@
     "    # Ingest\n",
     "    with tiledb.open(uri, \"w\") as tiledb_array:\n",
     "        idx = np.nonzero(data) if sparse else slice(None)\n",
-    "        tiledb_array[idx] = {\"features\": data[idx]}"
+    "        tiledb_array[idx] = data[idx]"
```

Comment:
You missed the respective schema change but apparently it still works, interesting.

Reply:
If I don't use it the sliced y is returned directly as `nd.array` and not as an `OrderedDict` inside the ingestor and thus breaks. We should probably need to support also unnamed attributes. Do you think that is ok to leave it like this for now?